website/docs/guides/mf-time-spine.md

```diff
@@ -0,0 +1,258 @@
+    columns:
+      - name: date_day
+        granularity: day
+```
+</File>
+
+This time spine YAML file configures the `time_spine` property so MetricFlow can use the table.
+
+## Run and test the time spine
+
+For the time spine table you created, let's run it and validate the output.
+
+1. Run the time spine model to create the table:
+   ```bash
+   dbt run --select time_spine_daily
+   ```
+
+2. Validate the output by querying the generated table:
+   ```sql
+   select * from your_project_schema.time_spine_daily
+   ```
+
+3. Check that the table:
+   - Contains one row per day.
+   - Covers the date range you want (5 years back to 30 days forward)
+
+4. (Optional) If you have [metrics](/docs/build/metrics) already defined in your project, you can query the table/metrics using [Semantic Layer commands](/docs/build/metricflow-commands) to validate the time spine. 
+   
+   Let's say you have a `revenue` metric defined. You can query the table/metrics using the following command:
+
+    ```bash
+    dbt sl query --metrics revenue --group-by metric_time
+    ```
+
+    This will output results similar to the following in the dbt Cloud IDE:
+    <Lightbox src="/img/quickstarts/dbt-cloud/validate-mf-timespine-output.jpg" title="Validate the time spine output in dbt Cloud IDE" />
+
+5. Double check that the results are correct and returning the expected data.
+
+## Add additional granularities (optional)
+
+To support multiple granularities (like hourly, monthly), create additional time spine tables and configure them in YAML.
+
+1. Add a new SQL file named `time_spine_hourly.sql` with the following content:
+<File name='models/marts/time_spine_hourly.sql'>
+
+```sql
+{{
+    config(
+        materialized = 'table',
+    )
+}}
+
+with hours as (
+
+    {{
+        dbt.date_spine(
+            'hour',
+            "to_date('01/01/2000','mm/dd/yyyy')",
+            "to_date('01/01/2025','mm/dd/yyyy')"
+        )
+    }}
+
+),
+
+final as (
+    select cast(date_hour as timestamp) as date_hour
+    from hours
+)
+
+select * from final
+-- filter the time spine to a specific range
+where date_hour > dateadd(year, -4, current_timestamp()) 
+  and date_hour < dateadd(day, 30, current_timestamp())
+
+```
+</File>
+
+2. Then update `_models.yml` file and add the hourly time spine (below the daily time spine config):
+
+<File name='_models.yml'>
+
+```yaml
+models:
+  - name: time_spine_daily
+    ... rest of the daily time spine config ...
+
+- name: time_spine_hourly
+    description: A time spine with one row per hour, ranging from 2000-01-01 to 2030-01-01.
+    time_spine:
+      standard_granularity_column: date_hour
+    columns:
+      - name: date_hour
+        granularity: hour
+```
+</File>
+
+3. Run the model to create the table:
+   ```bash
+   dbt run --select time_spine_hourly
+   ```
+
+4. Validate the output by querying the generated table:
+   ```bash
+   dbt sl query --metrics revenue --group-by metric_time__hour
```

Comment:
also the ide doesn't support `metric_time__hour` -- is this right?

Reply:
or should i scrap this section?

Reply:
I would think the IDE supports `metric_time__hour`, why do you say otherwise?

You could try 
`dbt sl list dimensions --metrics revenue`